frappe/desk/reportview.py

```diff
@@ -78,6 +78,12 @@ def execute(doctype, *args, **kwargs):
 	return DatabaseQuery(doctype).execute(*args, **kwargs)
 
 
+def get_distinct(distinct) -> str:
+	if distinct and frappe.conf.db_type == "mariadb":
```

Comment:
Why is this predicated on the db_type? That seems a bit out of context, condering that both are implementations of SQL and that we want to contrue a valid SQL (first of all) query.

Maybe there's another way or a different origin bug causing the behaviour?

This fix doesn't seem quite right, to me.